examples/tutorial/basic-5/tests/basic-5.ts

```diff
@@ -0,0 +1,116 @@
+import * as anchor from "@project-serum/anchor";
```

Comment:
Package namespace and version is outdated, use `@coral-xyz/anchor` from the repository instead.

Reply:
Done.